tasks/configure.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Configure New Relic PHP agent
+  become: yes
+  ini_file:
+    dest: "{{ sansible_newrelic_agent_php_ini_paths_base }}"
+    mode: 0644
+    section: "{{ item.section | default(None) }}"
+    option: "{{ item.option }}"
+    value: "{% if item.value == true %}true{% elif item.value == false %}false{% else %}{{ item.value }}{% endif %}"
```

Comment:
```suggestion
    value: "{{ 'true' if item.value == true else 'false' }}" 
```

Reply:
That removes the third case no right @wojtek-oledzki ?

Reply:
This was a suggestion from @dennisconrad, he's noticed that Ansible can often convert string representations of booleans behind the scenes and produce unexpected results.

Reply:
That's not the same unfortunatly, @wojtek-oledzki

Reply:
```
{{ 'true' if item.value == true else 'false' if item.value == false else item.value }}
```

Reply:
That's way less readable, I prefer the original :)